Tabloid/client/src/components/CategoryList.js

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from "react";
+import { deleteCategory, getAllCategories } from "../modules/categoryManager";
+import { Link, useHistory } from "react-router-dom";
+
+export const CategoryList = () => {
+  const [categories, setCategories] = useState([]);
+
+  const getCategories = () => {
+    getAllCategories().then((categories) => setCategories(categories));
+  };
+
+  const history = useHistory();
+
+  useEffect(() => {
+    getCategories();
+  }, []);
+
+  const handleDelete = (id) => {
+    deleteCategory(id)
+      .then(getCategories())
+      .then((c) => {
```

Comment:
Is this .then really necessary?

Reply:
Nope, it already gets taken care of elsewhere. Removed.